resources/views/components/nested-menu-dropdown.blade.php

```diff
@@ -0,0 +1,30 @@
+<div class="dropend">
```

Comment:
I would avoid creating a file in `components` that isn't an actual component... It would be confusing right?

Would it be too bad to copy-paste this in `menu-dropdown` itself?

Reply:
It worked for me... 🤷‍♂️ Check out https://github.com/Laravel-Backpack/theme-tabler/pull/143 please, I think it should be merged into this PR.